pkg/apis/proxy/v1alpha1/managedproxyconfiguration_types.go

```diff
@@ -219,6 +219,11 @@ type ManagedProxyConfigurationProxyServerEntrypoint struct {
 	// `hostname` points to a fixed hostname for serving agents' handshakes.
 	// +optional
 	Hostname *EntryPointHostname `json:"hostname"`
+
+	// `port` points to the port proxy-agent should access
```

Comment:
`port` is the target port to access proxy servers

Reply:
Comment udpated.